metadata/index.html

```diff
@@ -1678,7 +1696,10 @@
           Conversion specifications MUST define a default mapping from an annotated table that lacks any annotations (ie that is equivalent to an un-annotated table).
         </p>
         <p>
-          Conversion specifications MUST use either the <a href="#column-name"><code>name</code></a> or the <a href="#column-predicateUrl"><code>predicateUrl</code></a> of a column as the basis for naming machine-readable fields in the target format, such as the name of the equivalent element or attribute in XML, property in JSON or property URI in RDF.
+          Conversion specifications MUST use the <a>property value</a> of the <a href="#cell-aboutUrl"><code>aboutUrl</code></a> of the first cell column, if any, as the basis for identifying a mapped row.
+        </p>
+        <p>
+          Conversion specifications MUST use the <a>property value</a> of the <a href="#cell-propertyUrl"><code>propertyUrl</code></a> of a column as the basis for naming machine-readable fields in the target format, such as the name of the equivalent element or attribute in XML, property in JSON or property URI in RDF.
```

Comment:
The property value of `propertyUrl` is always defined using a default value if not defined explicitly. This allows the transformation specs to just use this value. Thus, if not specified, it will turn into `"{#_name}"`. Of course, a spec could decide to use the `name` property explicitly, which is also always defined to have a value of `"col.[N]"`.